resources/qml/components/UserListRow.qml

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: 2021 Nheko Contributors
+// SPDX-FileCopyrightText: 2022 Nheko Contributors
+// SPDX-FileCopyrightText: 2023 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import ".."
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import im.nheko 1.0
+
+ItemDelegate {
```

Comment:
this looks very similar to the AvatarListTile in resources/qml/components

Reply:
I didn't see that one because only the imagepack thing uses it -.- Are you sure it's suitable in its current form, and, if not, does it make sense to make it suitable? One problem I see is that I'd be dependent on buggy tap handlers to handle clicks, but I guess I could fix that. I'm not sure what other problems there might be. It just seems like my UserListRow is simpler, albeit possibly less versatile.

Reply:
I don't really care tbh. I just wanted to mention it, since you probably hadn't seen it.